src/OpenTelemetry/CHANGELOG.md

```diff
@@ -16,8 +16,13 @@
   ([#4375](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4375))
 
 * Tweaked the behavior of the `OpenTelemetryLoggerOptions.ParseStateValues`
-  flag, obsoleted `LogRecord.State` and `LogRecord.StateValues` properties, and
-  added `LogRecord.Body` and `LogRecord.Attributes` properties.
+  flag (attributes are now automatically included for most log messages by
+  default), obsoleted `LogRecord.State` and `LogRecord.StateValues` properties,
+  and added `LogRecord.Body` and `LogRecord.Attributes` properties.
+  ([#4334](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4334))
+
+* If a template (`{OriginalFormat}` attribute) cannot be found on log messages a
```

Comment:
We could make this more clear:

Tweaked the behavior of the `OpenTelemetryLoggerOptions.IncludeFormattedMessage`: `LogRecord.FormattedMessage` will now automatically be generated, if a template (`{OriginalFormat}` attribute) is not found on log messages (regardless of whether `OpenTelemetryLoggerOptions.IncludeFormattedMessage` is set to `true` or `false`).

Reply:
I just pushed a new version. LMK what you think.